### DISCUSSION TITLE
Support external temperature on thermostat card

### DISCUSSION THREAD

**Discussion by Wickedy123** [home-assistant/frontend#18970] [Dashboard]
The recent improvements (Dec-23) to the thermostat card are great and I'd love to use it, but they are no use for anyone who's heating set-up uses external temperature sensors, and I believe there are plenty of us. 

My TRVs (Danfoss in my case) completely ignore their own temperature sensor, so it makes no sense to use the standard thermostat card, which only displays this value (and the value is typically 2-3c higher).

The options out there for external sensors are very limited. Because I much prefer a round dial control I'm making do with Dark Thermostat Card, which was last updated in 2021. The only other option I'm aware of is Simple Thermostat, which allows sensors to be added. 

This issue was originally raised and dismissed back in 2021, which I think was a big mistake: https://github.com/home-assistant/frontend/issues/7036

Thank you for your consideration!

**Comment by frenck** [comment]
This is not a frontend issue, but a climate/backend issue.


As also stated in the same issue you linked: https://github.com/home-assistant/frontend/issues/7036#issuecomment-790698600

Please use our community forums to request features for that.

Thanks! 👍 

../Frenck

**Reply by Wickedy123** [reply]
Wow thanks for the swift answer - I've never understood that comment, can you elaborate? From the end user point of view, they want a configuration option to display the temperature for a different entity, instead of the climate's on-board sensor.

It doesn't make sense to me that climate devices themselves should have this configuration option, but instead it should just be possible to point the lovelace card at a different source (as is the case with Dark Thermostat and Simple Climate Card)

If you can help me understand, I will be happy to raise in the correct place.

**Reply by frenck** [reply]
A climate control processes the provided sensor to determine its havc action, which is not a frontend responsibility. The frontend display the climate entity as provided by the backend (which provides the current temperature that belong to these controls).

../Frenck

**Reply by Wickedy123** [reply]
Thanks for the info.. an 'end user' I can only express the requirement in terms of front end functionality and I can't comment on what the correct technical design should be beyond pointing out how I've seen it done in 3rd party cards. 

What I do know is that this is a bona fide requirement and the original issue should not have been closed in 2021 merely because it was raised in the wrong place - same for this ticket here. (Ideally there would be a place for people to raise general requirements without having to diagnose whether the solution should be 'back end' or 'front end')

I will raise this elsewhere and reference this thread. 

Thanks again @frenck for the very fast response!

**Reply by frenck** [reply]
>  should not have been closed in 2021 merely because it was raised in the wrong place 

It should, as it was in the wrong place.
